src/utils/Configuration.ts

```diff
@@ -420,7 +420,7 @@ export default class Configuration {
       Configuration.getConfig().Axios.timeout = Constants.AXIOS_DEFAULT_TIMEOUT;
     }
     if (Configuration.isUndefined(Configuration.getConfig().Axios.retries)) {
-      Configuration.getConfig().Axios.retries = 3;
+      Configuration.getConfig().Axios.retries = 0;
```

Comment:
Ouch. The default behaviour is with retry enabled, you can't disable retry globally, they make all the B2B communication work reliably (Concur, assets, etc.), you have to do it only in the unit tests.

Reply:
Concur provides its own configuration.
I think that the code that will call APIs with a retry should provide a specific configuration.
Maybe this is also related to a problem that I currently have with sending multiple time the same CDR to Gireve where we have locks in place.

Reply:
I doubt, the retry is triggered on very specific error types and the logs never expose retries done by Axios to Gireve. You simply can't disable a know to work default without ensuring that all callers have been audited first.

Reply:
> I doubt it, the retry is triggered on very specific error types and the logs never expose retries done by Axios to Gireve. You simply can't disable a know to work default without ensuring that all callers have been audited first.
> In the real world, you need to retry in most case on a REST API except for specific cases.

Yes you're right, it's deployed in our QA system.
I'll check today the OCPI part and Concur next.

Reply:
> ok. I'll revert the change and we will have to follow-up on thix one with a different PR.

No need to revert, we'll test it.

Reply:
The problem here is in the the way that global behavioural change is made: 
+ Hidden inside an unrelated PR; 
+ No team discussion on it triggered;
+ No rationale given.  

It really should have better managed if done otherwise with a kind of team implication.